community/tutorials/gdnative/gdnative-c-example.rst

```diff
@@ -0,0 +1,184 @@
+.. _doc_gdnative_c_example:
+
+GDNative C example
+==================
+
+Introduction
+------------
+This tutorial will introduce you to the bare minimum required to create GDNative modules. This should be your starting point into the world of GDNative, understanding the contents of this tutorial will help you in understanding all that is to come after this.
+
+Before we begin, you can download the source code to the sample object we'll be describing here by following this link:
+https://github.com/GodotNativeTools/GDNative-demos/tree/master/c/SimpleDemo
+
+This example project also contains a SConstruct file that makes compiling a little easier but in this tutorial we'll be doing things by hand.
+
+So what is GDNative and why was it added to Godot? GDNative is a framework added to Godot that lets us load dynamic libraries when we need to use them. The framework handles the loading and unloading of these libraries and the instantiation of objects contained within these libraries.
+It allows you to write extensions to Godot in either C, C++ or any other language bindings we may introduce in the future, compile them, use them, distribute them, etc. without needing to recompile Godot itself.
+
+Prerequisites
+-------------
+Before we start you'll need a few things.
+
+1) A Godot 3.0 executable, speaks for itself really
```

Comment:
I don't know if using "speak for itself" twice in such quick succession is a particularly good idea.